packages/plugin-board/src/board.js

```diff
@@ -127,6 +131,28 @@ const Board = SparkPlugin.extend({
     }, channel);
   },
 
+  _ensureOnlyConversationAuthorization(channel) {
+    // keep this function out of the critical path of creating a board
+    process.nextTick(() => {
+      if (this._isChannelCreatorAuthorized(channel.creatorId, channel.kmsMessage)) {
+        // remove channel creator from board service kms as we want to link to
+        // conversation KRO only
+        const kmsMessage = {
```

Comment:
I should've seen this last week, but can you implement this in kms.js? There are already test stubs for `addAuthorization` and `removeAuthorization`, I just never got around to writing them...